### ISSUE TITLE
Follow MVVM Architecture pattern

### ISSUE THREAD

**Issue by DroidPulkit** [DroidPulkit/TorontoWasteWizard#12]
Need to add ViewModels and Dagger2

**Comment by snufflesrea** [comment]
Hello, I am learning about architecture pattern currently. May I do this issue slowly? And may be can you guide where I can start?

**Comment by DroidPulkit** [comment]
Yeah sure, start by moving the logic from the Activity to the ViewModels, try and send a PR
